src/Services/Kros.Organizations.Api/Infrastructure/OrganizationRepository.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Kros.Organizations.Api.Domain;
+using Kros.KORM;
+using Kros.Utils;
+using System.Threading.Tasks;
+
+namespace Kros.Organizations.Api.Infrastructure
+{
+    /// <summary>
+    /// Repository for persistating <see cref="Organization"/>.
+    /// </summary>
+    public class OrganizationRepository: IOrganizationRepository
+    {
+        private IDatabase _database;
+
+        /// <summary>
+        /// Ctor.
+        /// </summary>
+        /// <param name="database">Database</param>
+        public OrganizationRepository(IDatabase database)
+        {
+            _database = Check.NotNull(database, nameof(database));
+        }
+
+        /// <inheritdoc />
+        public async Task CreateOrganizationAsync(Organization item)
+        {
+            var dbSet = _database.Query<Organization>().AsDbSet();
```

Comment:
If you have a new version of `KORM`, then you can simplify it `_database.AddAsync(item)`. See [documentation](https://github.com/Kros-sk/Kros.KORM#simplify-adding-deleting-and-editing-records).